ignite/metrics/regression/kendall_correlation.py

```diff
@@ -0,0 +1,109 @@
+from typing import Any, Callable, Tuple
+
+import torch
+
+from scipy.stats import kendalltau
```

Comment:
Let's put this import into `_compute_kendall_tau` method and in the constructor of `KendallRankCorrelation` let's check if scipy is present in the environment and raise an error if not like here:
https://github.com/pytorch/ignite/blob/aa3e3e13c214fe6cf72e941a46f13378911c8894/ignite/metrics/cohen_kappa.py#L64-L67

Reply:
Thank you for your review!
I addressed that.